processing/src/main/java/io/druid/query/MetricsEmittingQueryRunner.java

```diff
@@ -32,7 +32,7 @@
  */
 public class MetricsEmittingQueryRunner<T> implements QueryRunner<T>
 {
-  private static final String DEFAULT_METRIC_NAME = "query/time";
+  private static final String DEFAULT_METRIC_NAME = "query/portion/time";
```

Comment:
`portion` is a little bit vague, maybe we can be a bit more specific? Maybe `query/historical/time` and `query/realtime/time`?